### ISSUE TITLE
No Issues listed

### ISSUE THREAD

**Issue by WaynePhipps** [dhanasingh/redmine_wktime#26]
We're trying to use the plugin v1.8.1 with Redmine 3.0.3 on Postgresql.

We have a project with some issues open:
![current_issues](https://cloud.githubusercontent.com/assets/3416803/8279574/98048c28-18cd-11e5-8e8c-c4c788a1af95.png)

But no issues are shown in the list:
![issue list empty](https://cloud.githubusercontent.com/assets/3416803/8279591/b0b19928-18cd-11e5-853d-963dc552c5f3.png)

Any ideas how to resolve?

**Comment by suganya-thulasiraman** [comment]
We see  in your screenshot only 3 open features
1) Please check if the "Filter Issue by Tracker" is set to any other tracker(Bug, Support) in "Time & Expense" setting page.
2) And also please check the role of the user on the project "Phase 1" and check the "Issue Visibility" of the role in Administration -> Roles and Permission.
For more details, please refer http://www.adhisoftware.co.in/wp-content/uploads/TE-User-Guide.pdf

**Comment by WaynePhipps** [comment]
Thank you for your prompt response

Although in my case it didn't help, I was able to resolve the issue after some further digging.
I'll explain the issue I had for the benefit of others.

The problem appeared to be that the Issue dropdown was not populated when the Project dropdown was changed.
I tracked this down to the projectChanged call in edit.js so jumped onto the Javascript console to see what was happening.

It turned out that there was a 403 error accessing this file (amongst others) which had gone unnoticed:
![2015-06-22_1416](https://cloud.githubusercontent.com/assets/3416803/8286371/00bda046-1901-11e5-9f06-43b1ab253fa8.png)


After cd'ing into the public folder within Redmine, I ran a quick command to change the permissions:
find -type f -print0 | xargs -0 chmod o+r

This resolved the error.
